nix/targets/prod-build.nix

```diff
@@ -0,0 +1,66 @@
+#
+# This Nix expression builds an index.*.js file for the current repository given a node modules Nix expression
+#
+
+{ stdenv, stdenvNoCC, lib, target-os, callPackage, pkgs,
+  mkFilter, clojure, leiningen, maven, nodejs, localMavenRepoBuilder }:
+
+# The Nix expression takes a second argument to specify the node dependencies
+{ projectNodePackage }:
```

Comment:
Why is this a separate argument, and not part of the first?

Reply:
The first argument I pass when I import the package, and the second I pass when I actually invoke it.

Reply:
Makes sense. Than maybe update the comment? It's not a second argument. It's an argument to function that is returned from the Nix expression that is the file.